charts/velero/templates/serviceaccount-server.yaml

```
@@ -12,4 +12,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ include "velero.chart" . }}
+{{- if .Values.serviceAccount.server.labels }}
+{{ toYaml .Values.serviceAccount.server.labels | nindent 4 }}
+{{- end }}
```

nit: I'd prefer use `with` other than `if` when there is no `else` condition.
```suggestion
{{- with .Values.serviceAccount.server.labels }}
  {{- toYaml . | nindent 4 }}
{{- end }}
```

---

Done